rclcpp/test/rclcpp/test_executor.cpp

```diff
@@ -357,6 +366,14 @@ TEST_F(TestExecutor, spin_some_fail_add_handles_to_wait_set) {
   auto timer =
     node->create_wall_timer(std::chrono::milliseconds(1), [&]() {});
```

Comment:
this test was dependent on the subscription for `/parameter_events`, maybe this timer should be removed?